WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/Product+PriceSettingsViewModels.swift

```diff
@@ -9,11 +9,12 @@ extension Product {
         let currencyFormatter = CurrencyFormatter()
         let currencyCode = CurrencySettings.shared.currencyCode
         let unit = CurrencySettings.shared.symbol(from: currencyCode)
-        var value = currencyFormatter.formatAmount(regularPrice ?? "", with: unit) ?? "0"
+        var value = currencyFormatter.formatAmount(regularPrice ?? "", with: unit) ?? ""
         value = value.replacingOccurrences(of: unit, with: "")
         return UnitInputViewModel(title: title,
                                   unit: unit,
                                   value: value,
+                                  placeholder: "0",
```

Comment:
Do you think we should promote `"0"` into a constant somewhere? 🤔 Not sure which file it could be placed though.

Reply:
;) Done!